src/Tools/Microsoft.VisualStudio.Templates.AssemblyInfo.Wizard/AssemblyInfoTemplateWizard.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using EnvDTE;
+
+using Microsoft.VisualStudio.TemplateWizard;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+public partial class AssemblyInfoTemplateWizard : IWizard
+{
+    private static readonly string CSharp_Minimal_AssemblyInfoTemplate = $@"using System.Runtime.InteropServices;
```

Comment:
I assume we want both of these to be resource strings so they can be translated?

Reply:
I wasn't sure about localization ad the .editorconfig template has one line localized, but others not. Is this something the LOC team can handle? /cc @tmeschter

Reply:
I suggest extracting out the individual comments that need to be localized and putting those into resources. Then have the wizard inject those into the correct place in the code. For example, instead of

``` C#
// In SDK-style projects the assembly information is automatically
// generated on build.
```

Do this:

``` C#
$@"
...
// {Resources.AssemblyInformationAutomaticallyGeneratedComment}
...
";